test/fixtures/test-runner/timeout.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const test = require('node:test');
+const common = require('../../common');
+const fixtures = require('../../common/fixtures');
+
+test('should support timeout', async () => {
+  const stream = test.run({files: [
```

Comment:
```suggestion
test('should support timeout', async ({ signal }) => {
  const stream = test.run({
  signal,
  files: [
```

this will kill the zombie processes

Reply:
Updated thanks.